lib/alpha_taxonomy/taxon_link_deleter.rb

```diff
@@ -0,0 +1,27 @@
+module AlphaTaxonomy
+  class TaxonLinkDeleter
+    def initialize(logger: Logger.new(STDOUT))
+      @log = logger
+    end
+
+    def run!
+      base_paths = ENV.fetch("TAXON_CLEANUP").split(",")
```

Comment:
If you move this over to the rake task this class can be re-used elsewhere and you don't have to stub `ENV` in the tests.